app/_hub/kong-inc/aws-lambda/overview/_index.md

```diff
@@ -50,22 +50,22 @@ that the plugin now has the access permission configured in the target assumed r
 
 {% endif_plugin_version %}
 
-{% if_plugin_version gte:3.0.x %}
+{% if_plugin_version gte:2.8.x %}
```

Comment:
@windmgc I changed the if_version here since you said you backported this to 2.8. Let me know if that's incorrect and I can switch it back!

Reply:
@cloudjumpercat Thanks! You're correct, it should be greater than 2.8.x